sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -2232,7 +2236,7 @@ def post_port_mux_info_to_db(logical_port_name, mux_tbl, asic_index, y_cable_tbl
 
     for physical_port in physical_port_list:
 
-        if not y_cable_wrapper_get_presence(physical_port):
+        if not y_cable_wrapper_get_presence(physical_port) or cable_type == 'pseudo-cable':
```

Comment:
what is 'pseudo-cable'?

Reply:
this is a one-time call to make sure when the gRPC channels/stubs etc are created, ycabled posts the active-standby fields but only as `unknown/N/a`. The reason for doing it is ST today depends on having this fields populated albeit N/A or unknown, to post the full data to Kusto DB. 
this `pseudo-cable` logic takes care of it, and is only called once when Cable is either inserted/or ycabled spawns